samples/DFRobotDFPlayerMini/app/application.cpp

```diff
@@ -0,0 +1,47 @@
+using namespace std;
+#include <user_config.h>
+#include <SmingCore/SmingCore.h>
+#include <Libraries/DFRobotDFPlayerMini/DFRobotDFPlayerMini.h>
+
+#include <string>
```

Comment:
Are the include string and include string.h really needed in this sample? If not, please remove them.

Reply:
Unwanted code removed.